README.md

```diff
@@ -37,11 +37,24 @@ By default, this will run tests. If you want to skip running tests use
 
     $ mvn install -DskipTests
 
+The build is stable starting with Maven version `3.5.x` or higher, which was 
+when the [usage of the `revision` property](https://maven.apache.org/maven-ci-friendly.html) was introduced. 
+With a lower version you will get errors when trying to resolve dependencies created in the Antenna build,
+since the version is not correctly parsed. A possible error message would look like this:
+
+```  
+Failed to process POM for org.eclipse.sw360.antenna:model:jar:1.0.0-SNAPSHOT: 
+Non-resolvable parent POM for org.eclipse.sw360.antenna:model:1.0.0-SNAPSHOT: 
+Could not find artifact org.eclipse.sw360.antenna:antenna-management:pom:${revision} 
+in central (https://repo.maven.apache.org/maven2/) and 'parent.relativePath' points at wrong local POM
+[ERROR] org.eclipse.sw360.antenna:model:jar:1.0.0-SNAPSHOT
+```
+
 #### Optional Profiles
 You can activate the following optional profiles:
 - `-P integration-test`: activates also the optional profile for integration testing in the sw360 module (see also `./modules/sw360/README.md`)
 - `-P site-tests`: which activates the site tests in `./documentation/`
-
+https://github.com/eclipse/antenna/pull/567#pullrequestreview-461588292
```

Comment:
This change probably slipped in by accident I assume.

Reply:
yeah, don't know how that got in there, tbh. Thanks for spotting it.